packages/vue/src/index.ts

```diff
@@ -34,9 +34,11 @@ export const useForm = <Data extends Record<string, unknown>>(method: RequestMet
             form.validating = validator.validating()
         })
         .on('validatedChanged', () => {
+            // @ts-expect-error Validator does not know shape of data.
```

Comment:
This was an error for me on build.

I don't understand why it wasn't an error for `main`.